src/probabilities_estimators/histograms/rectangular_binning.jl

```diff
@@ -15,35 +18,113 @@ Binning instructions are deduced from the type of `ϵ` as follows:
     intervals that cover all data.
 4. `ϵ::Vector{Float64}` divides the i-th coordinate axis into intervals of fixed size
     `ϵ[i]`, starting from the axis minima until the data is completely covered by boxes.
+
+!!! note "Be aware when mixing methods"
+    If the range of the data along some dimension is `[r1, r2]`, then
+    `RectangularBinning(N)` results in a bin edge length of
+    `nextfloat(r2 - r1) / N)`. `nextfloat` is used to ensure data are completely
+    covered. Keep this in mind if you're switching between the `ϵ::Int` and `ϵ::Float64`
+    ways of constructing grids.
```

Comment:
I don't understand what this comment is trying to achieve. I think the goal is to educate on how the binning works if `n` is Int. But then, why not just state exactly what we do in the 1. bullet point, which would save us 3 lines of documentation? Also, I thought we take the nextfloat twice in the source code.

Reply:
If I remember correctly, it originated back when using an integer number of bins gave a different number of bins that using exact bin sizes. But this is fixed now. An honestly, upon reading this comment now, it doesn't make much sense. I think we can just remove this comment.